test/kafka-exactly-once/after-restart.td

```diff
@@ -77,29 +77,29 @@ $ kafka-ingest format=avro topic=input schema=${schema} timestamp=14
 {"before": null, "after": {"row": {"a": 4, "b": 1}}}
 {"before": null, "after": {"row": {"a": 5, "b": 2}}}
 
-$ kafka-verify format=avro sink=materialize.public.output_byo sort-messages=true
+$ kafka-verify format=avro sink=materialize.public.output_byo topic=output-byo-sink-${testdrive.seed} nonced-topic=false sort-messages=true
```

Comment:
These technically wouldn't need the `nonced-topic=false`, because the default is `false`, right?

Reply:
Yes! I can remove all the `nonced-topic=false` flags... I think they were more for me anyway :joy